src/database/realtime/polling/PacketReceiver.ts

```diff
@@ -45,7 +47,6 @@ export class PacketReceiver {
       }
       if (this.currentResponseNum === this.closeAfterResponse) {
         if (this.onClose) {
-          clearTimeout(this.onClose);
```

Comment:
I removed this `clearTimeout` since `this.onClose` is a function. Probably left over from a previous change.